radixdlt-core/radixdlt/src/main/java/com/radixdlt/consensus/UnverifiedVertex.java

```diff
@@ -155,7 +163,7 @@ public static UnverifiedVertex create(
 		List<Txn> txns,
 		BFTNode proposer
 	) {
-		Objects.requireNonNull(qc);
+		requireNonNull(qc);
```

Comment:
This is not necessary as this validation happens in the constructor.

Reply:
done